assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/TotalUserCount.js

```diff
@@ -73,8 +74,19 @@ export default function TotalUserCount( { dimensionName, dimensionValue } ) {
 	const error = useSelect( ( select ) => select( STORE_NAME ).getErrorForSelector( 'getReport', [ args ] ) );
 	const report = useSelect( ( select ) => select( STORE_NAME ).getReport( args ) );
 
-	if ( ! loaded || error || isZeroReport( report ) ) {
-		// The UserCountGraph component will return appropriate PreviewBlock/ReportError/ReportZero component
+	if ( ! loaded ) {
+		return (
+			<PreviewBlock
+				className="googlesitekit-widget--analyticsAllTrafficV2__totalcount--loading"
+				width="220px"
+				height="130px"
+				shape="square"
+			/>
+		);
+	}
```

Comment:
I don't think we need to modify this component here. The original idea was to include a single loader / `ReportZero` / `ReportError` because `TotalUserCount` and `UserCountGraph` should be displayed as one - basically `TotalUserCount` always returns `null` for those scenarios while `UserCountGraph` would return the actual component for it. See https://github.com/google/site-kit-wp/pull/2589#discussion_r553093002. Is there a specific reason why you added the `PreviewBlock` here?

Reply:
These reports are slightly different: one requests the total user count and the other requests users for each day. This means they resolve at different times, and putting the loading state for another component inside this one seems odd to me. If they should be anywhere shared it should be the parent component I think.

But in my usage these components can exit "loading" state at different times, so they shouldn't share a loading state. This results in one disappearing and the layout "jumping" around while one is loaded and another isn't.

I suppose we could consolidate the API requests but the total user API request does not include the same data as the new one for the daily users report. Again: combining the numbers from the daily report could give us the user count and reduce API requests which would be good, but I wasn't sure if we wanted to do that now or later 🙂 

Thoughts?

Reply:
I originally thought we would use the same API request for both, but it's okay not to do that for now. We definitely _can_ do it:
* Request with compare dates (needed for `TotalUserCount`), then in `UserCountGraph` only consider the second half of rows (the "main" date range).
* Request with `ga:date` dimension (needed for `UserCountGraph`), which is irrelevant for `TotalUserCount` since it only looks at the totals anyway.

Maybe worth a follow-up issue.